com.github.Ryujinx.json

```diff
@@ -0,0 +1,1266 @@
+{
+    "app-id": "com.github.Ryujinx",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
```

Comment:
Update runtime to 20.08
```suggestion
    "runtime-version": "20.08",
```

Reply:
@LeandroStanger You haven't tested your change, have you? There's no dotnet extension for 20.08 yet. Please don't suggest anything you did not try.

Reply:
Still not dotnet extension